<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KoDocumentInfoAboutWidget</class>
 <widget class="QWidget" name="KoDocumentInfoAboutWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>443</width>
    <height>546</height>
   </rect>
  </property>
  <layout class="QGridLayout">
   <item row="10" column="1">
    <widget class="QLabel" name="lblModified">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="11" column="0">
    <widget class="QLabel" name="lblLastPrintedDesc">
     <property name="text">
      <string>Last printed:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QLabel" name="filePathLabel">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="10" column="0">
    <widget class="QLabel" name="lblModifiedDesc">
     <property name="text">
      <string>Modified:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="Line" name="filePathSeparatorLine">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="3">
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="lblLicense">
     <property name="text">
      <string>License/Rights:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="5" column="1" colspan="2">
    <widget class="QLineEdit" name="leLicense"/>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="lblTypeDesc">
     <property name="text">
      <string>Type:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="lblKeywordsDesc">
     <property name="text">
      <string>&amp;Keywords:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>leKeywords</cstring>
     </property>
    </widget>
   </item>
   <item row="13" column="0">
    <widget class="QLabel" name="lblRevisionDesc">
     <property name="text">
      <string>Revision number:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="2">
    <widget class="QLineEdit" name="leSubject"/>
   </item>
   <item row="4" column="1" colspan="2">
    <widget class="QLineEdit" name="leKeywords">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QLineEdit" name="leTitle"/>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="lblCreatedDesc">
     <property name="text">
      <string>Created:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="12" column="0">
    <widget class="QLabel" name="lblEditingDesc">
     <property name="text">
      <string>Total editing ti&amp;me:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>pbReset</cstring>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="lbCommentsDesc">
     <property name="text">
      <string>Description:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
     </property>
     <property name="buddy">
      <cstring>meDescription</cstring>
     </property>
    </widget>
   </item>
   <item row="12" column="1">
    <widget class="QLabel" name="lblEditing">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="filePathInfoLabel">
     <property name="text">
      <string>Path:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <widget class="QLabel" name="lblCreated">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="12" column="2">
    <widget class="QPushButton" name="pbReset">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="text">
      <string>&amp;Reset</string>
     </property>
    </widget>
   </item>
   <item row="13" column="1">
    <widget class="QLabel" name="lblRevision">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="6" column="1" colspan="2">
    <widget class="QTextEdit" name="meDescription"/>
   </item>
   <item row="8" column="1">
    <widget class="QLabel" name="lblType">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="11" column="1">
    <widget class="QLabel" name="lblLastPrinted">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lblTitleDesc">
     <property name="text">
      <string comment="Document title">Title:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>leTitle</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="lblSubjectDesc">
     <property name="text">
      <string>Sub&amp;ject:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>leSubject</cstring>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
